InterfacePython/example_optimization_01.py

```diff
@@ -28,12 +28,70 @@
 # IN WHOLE OR IN PART THE USE, STORAGE OR DISPOSAL OF THE SOFTWARE.
 ##############################################################################################################################################################################
 
-import Tasmanian
-import numpy
+from Tasmanian import Optimization as Opt, DREAM
+import TasmanianSG as SG
```

Comment:
Do not import TasmanianSG, it's a deprecated way of importing things.

All TasmanianSG definitions are under Tasmanian.SparseGrid.

Reply:
Changed. I now explicitly import the functions and libraries from `Tasmanian` that I need.